packages/proxy/src/proxy.ts

```diff
@@ -187,11 +198,29 @@ export async function proxyV1({
   }
 
   // Caching is enabled by default, but let the user disable it
-  const useCacheMode = parseEnumHeader(
+  // XXX is there a clean way to specify "auto" caching behavior using only cache-control directives?
```

Comment:
i think i like the way you've currently done it, in which we default to "auto" and "always" requires the explicit header